sdk/containerregistry/ci.yml

```diff
@@ -30,6 +30,7 @@ extends:
   parameters:
     TestProxy: true
     ServiceDirectory: containerregistry
+    ValidateFormatting: true
```

Comment:
I have tried to use `black -l 120` to format mgmt SDK code but CI still fails. Maybe there is still difference. Now that you have formatted `azure-containerregistry`, there is no need to check it in CI.

Reply:
Thanks for your update! `setup.py` is not blacked, I just pushed the updates, hope ci will pass. Btw, here's how we run black: https://github.com/Azure/azure-sdk-for-python/blob/main/doc/eng_sys_checks.md#black